salt/config/__init__.py

```diff
@@ -395,9 +395,9 @@
     'return_retry_timer': int,
     'return_retry_timer_max': int,
 
-    # Specify a returner in which all events will be sent to. Requires that the returner in question
-    # have an event_return(event) function!
-    'event_return': str,
+    # Specify one or more returners in which all events will be sent to. Requires that the returners
+    # in question have an event_return(event) function!
+    'event_return': list,
```

Comment:
This will cause usage of a single ``event_return`` (i.e. a string and not a list) to log warnings about invalid data types. Given that the default value for ``event_return`` is an empty string, this would not be good as it would cause a lot of spurious warnings in the log.

This should be changed to:

```python
'event_return': (list, string_types),
```